### PR TITLE
Updates build to package Akka.Streams, Akka.Persistence.Query

### PR DESCRIPTION
**Pull request by cconstantin**
Temporarily includes System.Reactive.Streams

### CODE REVIEW

File: src/core/Akka.Persistence.Query/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,17 +1,20 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
+﻿//-----------------------------------------------------------------------
+// <copyright file="AssemblyInfo.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
```
Comment (Silv3rcircl3) [inline src/core/Akka.Persistence.Query/Properties/AssemblyInfo.cs]: It's now called lightbend

File: src/core/Akka.Streams/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,17 +1,21 @@
-﻿using System.Reflection;
+﻿//-----------------------------------------------------------------------
+// <copyright file="AssemblyInfo.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
```
Comment (Silv3rcircl3) [inline src/core/Akka.Streams/Properties/AssemblyInfo.cs]: As above

File: src/System.Reactive.Streams/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -1,5 +1,4 @@
 ﻿using System.Reflection;
```
Comment (Silv3rcircl3) [inline src/System.Reactive.Streams/Properties/AssemblyInfo.cs]: Copyright header is missing

File: src/core/Akka.Persistence.Query/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (cconstantin) [inline src/core/Akka.Persistence.Query/Properties/AssemblyInfo.cs]: None of the copyright notices have been updated, I assume they would all be updated in one shot?

**Pull request merged**